guestbook/records/models.py

```diff
@@ -0,0 +1,18 @@
+from django.db import models
+from django.utils import timezone
+
+
+# Create your models here.
+class Record(models.Model):
+    class Meta:
```

Comment:
The recommendation in Django itself says to put `class Meta` after the fields: https://docs.djangoproject.com/en/1.8/internals/contributing/writing-code/coding-style/#model-style but that's up for everybody to decide on their own.